test/test_common/printers.h

```diff
@@ -11,34 +11,34 @@ namespace Http {
  * Pretty print const HeaderMapImpl&
  */
 class HeaderMapImpl;
-void PrintTo(const HeaderMapImpl& headers, std::ostream* os);
+void printTo(const HeaderMapImpl& headers, std::ostream* os);
```

Comment:
I think these are needed to stay `PrintTo` because it is used by [googletest](https://chromium.googlesource.com/external/github.com/google/googletest/+/HEAD/googletest/include/gtest/gtest-printers.h#37).
Maybe it will be possible to add a no-linter directive here.

Reply:
nice catch, done